### PR TITLE
Update singular noun treatment of products

### PR DESCRIPTION
**Pull request by abschu**
The handbook doesn't reflect the reality of how we treat products.

@iskyOS @malomarrec - can you think of an example to add where batch changes would be lowercase and plural (when referring to taking an action/generic batch changes)?

### CODE REVIEW

File: content/company-info-and-process/communication/content_guidelines/style_and_mechanics.md [hunk 1]
```diff
@@ -854,7 +854,7 @@ List of product names:
 - With Sourcegraph Universal Code Search you can...
 - You can search across all your repositories with Universal Code Search.
 - Companies benefit from a universal code search tool because...
-- Sourcegraph Batch Changes are... Batch Changes allow you to...
+- Sourcegraph Batch Changes is... Batch Changes allows you to...
 - To create a batch change...
```
Comment (iskyOS) [inline content/company-info-and-process/communication/content_guidelines/style_and_mechanics.md:858]: @abschu, one potential way to have "batch changes" per your comment:

```suggestion
- To create a batch change...
- You can create batch changes for various purposes, such as upgrading dependencies across...
```

**Review by iskyOS — COMMENTED** [review]
Left one suggestion for an addition, otherwise LGTM!

File: content/company-info-and-process/communication/content_guidelines/style_and_mechanics.md (hunk 1 above)
Comment (malomarrec) [reply content/company-info-and-process/communication/content_guidelines/style_and_mechanics.md:858]: I would use
> such as changing an API call site across many repositories

**Review by malomarrec — APPROVED** [review]

**Review by abschu — COMMENTED** [review]
Reviewed

**Pull request merged**